cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/contactpoint/HttpClusterBootstrapRoutes.scala

```diff
@@ -72,6 +72,6 @@ object ClusterBootstrapRequests {
   import akka.http.scaladsl.client.RequestBuilding._
 
   def bootstrapSeedNodes(baseUri: Uri): HttpRequest =
-    Get(baseUri + "/bootstrap/seed-nodes")
+    Get(baseUri.toString + "/bootstrap/seed-nodes") // FIXME is there a better way than string concatenation?
```

Comment:
@johanandren do you know? was the previous `+` also string concatenation or some implicit from akka-http?